src/components/checkboxes/index.md.njk

```diff
@@ -11,6 +11,8 @@ layout: layout-pane.njk
 
 Let users select one or more options by using the checkboxes component.
 
+Additional test paragraph, with more detail.
```

Comment:
Nice, that's better.

What do you think about changing the wording slightly so it's more like this?

```suggestion
New test paragraph, with detail so it makes sense.
```